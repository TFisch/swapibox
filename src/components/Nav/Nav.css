.nav-wrapper {
  background-color: magenta;
  height: 10em;
  margin: auto 0;
  width: 100%; }

.header-row {
  width: 100%; }

.button-row {
  width: 100%; }

.title-logo {
  font-size: 3em;
  color: lime; }
