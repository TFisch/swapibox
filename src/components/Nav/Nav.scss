@import "../../styles/mixins.scss";
@import "../../styles/variables.scss";


.nav-wrapper {
  background-color: $color-four;
  height: 10em;
  margin: auto 0;
  width: 100%;
  z-index: 1; position:relative;
}

.header-row {
  @include flexBetween();
  align-items: center;
  margin-left: 39em;
  width: 58em;
}

.button-row {
  @include flexBetween();
  align-items: center;
  margin-left: 35em;
  width: 25em;
}

.title-logo {
  border-bottom: 3px solid $color-gold;
  color: $color-one;
  font-family: $font-fam;
  font-size: 3em;
  margin-top: .4em;
  padding: 6px;
  text-shadow: 2px 2px $color-three;
}

.fav-counter {
  border: 3px solid $color-gold;
  color: $color-one;
  width: 1.5em;
}

.fav-row {
  align-items: center;
  display: flex;
  justify-content: center;
  margin: auto 0;
  padding: 0;
  width: 30em;
}
