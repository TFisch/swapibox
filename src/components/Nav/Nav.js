import React, { Component } from 'react';
import './Nav.css';

class Nav extends Component {
  constructor() {
    super()
  }

  render() {
    return (
      <div className="nav-wrapper">
        <div className="header-row">
          <h1 className="title-logo">SWAPIBox</h1>
          <span className="fav-row">
            <div className="favorites">FAV PLACEHOLDER</div>
            <div className="fav-counter">0</div>
          </span>
        </div>
        <div className="button-row">
          <div className="favorites people">PEOPLE</div>
          <div className="favorites planets">PLANETS</div>
          <div className="favorites vehicles">VEHICLES</div>
        </div>
      </div>
    )
  }
}

export default Nav;