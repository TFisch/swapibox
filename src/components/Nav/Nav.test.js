import React from 'react';
import { shallow } from 'enzyme';
import Nav from '../Nav/Nav';

