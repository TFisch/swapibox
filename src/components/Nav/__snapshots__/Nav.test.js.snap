// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nav Should match snapshot 1`] = `"<div class=\\"nav-wrapper\\"><div class=\\"header-row\\"><h1 class=\\"title-logo\\">SWAPI-Box</h1><span class=\\"fav-row\\"><button class=\\"FAVORITES\\" value=\\"FAVORITES\\">FAVORITES</button><div class=\\"fav-counter\\"></div></span></div><div class=\\"button-row\\"><button class=\\"PEOPLE\\" value=\\"PEOPLE\\">PEOPLE</button><button class=\\"VEHICLES\\" value=\\"VEHICLES\\">VEHICLES</button><button class=\\"PLANETS\\" value=\\"PLANETS\\">PLANETS</button></div></div>"`;
